.rubocop.yml

```diff
@@ -144,3 +144,5 @@ RSpec/Rails/HaveHttpStatus:
   Enabled: true
 RSpec/Rails/InferredSpecType:
   Enabled: true
+RSpec/Capybara/MatchStyle:
+  Enabled: true
```

Comment:
This might be redundant, as we don't use Capybara in our code. I've removed the rest unrelated cops in #1475.

Reply:
Oops indeed it is. I removed it.